src/index.html

```diff
@@ -4,9 +4,377 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo_link">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__image">
```

Comment:
```suggestion
        <img
          src="images/logo.png"
          alt="logo"
          class="header__image"
        >
```
Располагай открывающую и закрывающую скобки на одном уровне